sdk/typescript/test/bcs/bcs.test.ts

```diff
@@ -3,13 +3,12 @@
 
 import { BCS } from '../../src/bcs';
 import { Base64DataBuffer as B64 } from '../../src';
-// import { HexDataBuffer as HEX } from '../../src';
-import { BN } from 'bn.js';
+import * as BN from 'bn.js';
```

Comment:
nit: i didn't see anything else being used from BN, we could make this import more specific to avoid having to do `new BN.BN(1)`